src/settings.ts

```diff
@@ -80,19 +80,19 @@ if (process.env.BLOCKCHAIN_URL) {
 if (env !== 'test') console.log('settings', settings);
 
 settings.secrets = {
-  createAccountPrivateKey: EosioUtil.defaultAntelopePrivateKey.toString(),
+  createAccountPrivateKey:
+    'PVT_K1_24kG9VcMk3VkkgY4hh42X262AWV18YcPjBTd2Hox4YWoP8vRTU',
```

Comment:
here why we didn't use from env?

Reply:
we are using from env. this is just the default. See https://github.com/Tonomy-Foundation/Tonomy-Communication/pull/96/files#diff-bb57a3bd912abc3ec2e729cb8a743838487677a5517683d0c8913a3619ac296aR95